l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -868,6 +868,7 @@ def generate_attach_report(self, inv):
                 'type': 'binary'
             }
             attachment = attachment_model.create(data_attach)
+        inv.write({'fatturapa_doc_attachments': [(5, )]})
```

Comment:
Come mai doppio? Quando viene creato 2 volte?

Reply:
Allora l'allegato in `fatturapa_doc_attachments` che contiene il PDF dovrebbe essere eliminato quando viene eliminato l'XML. Lo puoi identificare con `is_pdf_invoice_print`. Vedi `_compute_has_pdf_invoice_print`

Reply:
@eLBati (della serie meglio tardi che mai) intendi così?